cmd/apply.go

```diff
@@ -317,42 +350,49 @@ func process(plan types.Plan, prefs InstallPreferences) error {
 		sealedSecretsErr := installSealedSecrets()
 		if sealedSecretsErr != nil {
 			log.Println(sealedSecretsErr)
-		}
-
-		for i := 0; i < retries; i++ {
-			log.Printf("Are SealedSecrets ready? %d/%d\n", i+1, retries)
-			ready := sealedSecretsReady()
-			if ready {
-				break
-			}
-			time.Sleep(time.Second * 2)
+			return sealedSecretsErr
 		}
 
 		pubCert := exportSealedSecretPubCert()
 		writeErr := ioutil.WriteFile("tmp/pubcert.pem", []byte(pubCert), 0700)
 		if writeErr != nil {
 			log.Println(writeErr)
+			return writeErr
 		}
 	}
 
-	cloneErr := cloneCloudComponents(plan.OpenFaaSCloudVersion)
+	cloneErr := cloneCloudComponents(plan.OpenFaaSCloudVersion, additionalPaths)
 	if cloneErr != nil {
 		return cloneErr
 	}
 
-	deployErr := deployCloudComponents(plan)
+	deployErr := deployCloudComponents(plan, additionalPaths)
 	if deployErr != nil {
 		return deployErr
 	}
 
 	return nil
 }
 
-func helmRepoUpdate() error {
-	log.Println("Updating helm repos")
+func buildPath(additionalPaths []string) string {
+
+	pathVal := "PATH="
+	for _, p := range additionalPaths {
+		pathVal = pathVal + p + ":"
+	}
+
+	// pathVal = strings.TrimRight(pathVal, ":")
```

Comment:
if you are committing again it would be nice to get rid of this.

Reply:
Thanks I'll fix that.